README.md

```diff
@@ -10,14 +10,28 @@ Decompression Membership system is website that allows you to have event pages a
 
 To run this application locally for development, you will need to install:
 
-- ruby 2.7.6 (I recommend `rvm`, but you could use `rbenv` or any other technique. The version is specified in `.ruby-version`)
-- Postgresql (I use Postgres.app on OSX)
+- Ruby 3.3.0
```

Comment:
I'm not sure why this commit is part of this PR, and it also includes a lot of other changes to the README that aren't associated to mailcatcher, but they all look OK to me, so just 🏄‍♂️

Reply:
I can split this into a separate PR if needed